hazelcast/src/main/java/com/hazelcast/internal/diagnostics/DiagnosticsLogWriter.java

```diff
@@ -93,6 +93,13 @@ protected DiagnosticsLogWriter write(long i) {
         return this;
     }
 
+    protected DiagnosticsLogWriter writeDate(long epochTimeMs) {
+        date.setTime(epochTimeMs);
+        appendDateTime();
```

Comment:
It is intended. The object is mutable and shared between processing to prevent litter. The writers are single threaded; so no problem.